### DISCUSSION TITLE
Is it possible for an EDC Connector and a Dataspace Connector to communicate with each other?

### DISCUSSION THREAD

**Discussion by DimitrisGk-iel** [eclipse-edc/.github#30] [Q&A] [answered]
When initiating the communication from the EDC Connector towards the Dataspace Connector I get the following log from the latter:

> 2023-05-23T12:12:21,699 [https-jsse-nio-8080-exec-8] INFO - Received incoming message. [code=(IMSMEI0059)],
2023-05-23T12:12:21,700 [https-jsse-nio-8080-exec-8] INFO - Incoming message header: {
	"ids:modelVersion": "4.1.3",
	"ids:senderAgent": {
		"@id": "urn:connector:provider"
	},
	"ids:securityToken": {
		"ids:tokenValue": "{\"region\":\"eu\",\"audience\":\"https://dataspaceconnector:8080/api/ids/data\",\"clientId\":\"anonymous\"}",
		"ids:tokenFormat": {
			"@id": "https://w3id.org/idsa/code/JWT",
			"@type": "ids:TokenFormat"
		},
		"@id": "https://w3id.org/idsa/autogen/dynamicAttributeToken/d2d893b7-a177-40e7-9b68-764ca215f8bd",
		"@type": "ids:DynamicAttributeToken"
	},
	"ids:issued": {
		"@value": "2023-05-23T15:12:21.618+03:00",
		"@type": "http://www.w3.org/2001/XMLSchema#dateTimeStamp"
	},
	"ids:issuerConnector": {
		"@id": "urn:connector:provider"
	},
	"ids:recipientConnector": [
		{
			"@id": "https://dataspaceconnector:8080/api/ids/data"
		}
	],
	"@id": "https://w3id.org/idsa/autogen/descriptionRequestMessage/327e5ae0-35dd-4756-9e3b-6d9669b1674e",
	"@context": "https://w3id.org/idsa/contexts/context.jsonld",
	"@type": "ids:DescriptionRequestMessage",
	"querySpec": {
		"offset": 0,
		"limit": 50,
		"sortOrder": "ASC"
	}
} [code=(IMSMEI0060)],
2023-05-23T12:12:21,701 [https-jsse-nio-8080-exec-8] WARN - Infomodel model version validation of received messages is switched on. Model-version of incoming message not supported. Sending BAD_REQUEST response as a result. [code=(IMSMEW0042), response-message=(Infomodel version of incoming Message not in supported inbound model version list! [incoming=(4.1.3), supported=([4.2.6, 4.2.7, 4.2.0, 4.2.1, 4.1.2, 4.0.0, 4.1.0, 4.2.4, 4.2.5, 4.2.2, 4.2.3])])]

Is the information model version of the EDC Connector a limitation for establishing a communication with other connectors? 
Have you ever tried/succeeded In establishing a communication with another third party connector accept the EDC one?

**Comment by juliapampus** [comment, marked as answer]
Please see eclipse-edc/Connector#1549. The EDC are implementing the asynchronous [Dataspace Protocol](https://github.com/International-Data-Spaces-Association/ids-specification) whereas the DSC is implementing synchronous [IDS multipart messages](https://international-data-spaces-association.github.io/DataspaceConnector/Documentation/v6/Messages).
